### PR TITLE
[6.15.z] CV already published + single LCE

### PR DESCRIPTION
**Pull request by lhellebr**
Fix the test. CV is already published and promoted by setup_org_for_a_rh_repo. Also, don't use 2 LCEs needlessly.

### CODE REVIEW

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_host.py::test_negative_without_attach_with_lce

**Comment by lhellebr** [comment]
I doubt the CQ failure is related
```
FAILED tests/robottelo/test_shared_resource.py::test_shared_resource_multithreading - AssertionError: assert not True
```

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 5838
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_host.py::test_negative_without_attach_with_lce
Test Result : ================== 2 passed, 46 warnings in 839.95s (0:13:59) ==================
```

**Review by pondrejk — APPROVED** [review]

**Review by pnovotny — APPROVED** [review]

File: tests/foreman/cli/test_host.py [hunk 1]
```diff
@@ -2012,22 +2012,31 @@ def test_negative_without_attach(
...
 
-    :expectedresults: repository not enabled on host
+    :expectedresults: Repository enabled with SCA. Failure without SCA.
 
     :parametrized: yes
     """
+    if sca_enabled:
+        function_org.sca_enable()
+    else:
+        function_org.sca_disable()
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_host.py:2039]: As SCA will be enabled as default here, so we won't need to enable here 
```suggestion
    if not sca_enabled:
        function_org.sca_disable()
```

File: tests/foreman/cli/test_host.py [hunk 2]
```diff
@@ -2046,27 +2055,22 @@ def test_negative_without_attach_with_lce(
...
-    )
-    pool_id = subscriptions.stdout.strip()
-    # attach to plain RHEL subsctiption
-    host_subscription_client.subscription_manager_attach_pool([pool_id])
     assert host_subscription_client.subscribed
-    host_subscription_client.enable_repo(REPOS['rhst7']['id'])
+    res = host_subscription_client.enable_repo(REPOS['rhst7']['id'])
+    if sca_enabled:
+        assert res.status == 0
+        assert f"Repository '{REPOS['rhst7']['id']}' is enabled for this system."
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_host.py:2070]:
```suggestion
        assert f"Repository '{REPOS['rhst7']['id']}' is enabled for this system." not in res.stdout
```

File: tests/foreman/cli/test_host.py [hunk 3]
```diff
@@ -2046,27 +2055,22 @@ def test_negative_without_attach_with_lce(
...
+    else:
+        assert res.status != 0
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_host.py:2072]: Can we check for specific RC here, RC=1 probably?

File: tests/foreman/cli/test_host.py [hunk 4]
```diff
@@ -2046,27 +2055,22 @@ def test_negative_without_attach_with_lce(
...
-    # get list of available subscriptions which are matched with default subscription
-    subscriptions = host_subscription_client.run(
-        f'subscription-manager list --available --matches "{DEFAULT_SUBSCRIPTION_NAME}" --pool-only'
-    )
-    pool_id = subscriptions.stdout.strip()
-    # attach to plain RHEL subsctiption
-    host_subscription_client.subscription_manager_attach_pool([pool_id])
     assert host_subscription_client.subscribed
-    host_subscription_client.enable_repo(REPOS['rhst7']['id'])
+    res = host_subscription_client.enable_repo(REPOS['rhst7']['id'])
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_host.py:2067]:
```suggestion
    res = host_subscription_client.enable_repo(REPOS['rhsclient7']['id'])
```

File: tests/foreman/cli/test_host.py (hunk 1 above)
Comment (lhellebr) [reply tests/foreman/cli/test_host.py:2039]: I'd prefer to keep this as it is, in case it is not enabled by default (perhaps in future z-stream).

**Comment by lhellebr** [comment]
Gaurav's comments addressed

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_host.py::test_negative_without_attach_with_lce

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 5924
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_host.py::test_negative_without_attach_with_lce --external-logging
Test Result : ================== 2 passed, 47 warnings in 862.52s (0:14:22) ==================
```

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**